### PR TITLE
Add metadata to examples/models/file/daylight.py (#11765)

### PR DESCRIPTION
**Pull request by ramanshah**
This adds a metadata block for one example script among the Bokeh examples.

- [X] issues: addresses #11765

### CODE REVIEW

File: examples/models/file/daylight.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
+''' A plot shading the daylight hours in a city's local time (which are
```
Comment (mattpap) [inline examples/models/file/daylight.py:1]: "shading" seems to me an uncommon wording in this context. Maybe use "showing"?

File: examples/models/file/daylight.py [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+discontinuous due to summer time) using the Patch and Line plot elements to
```
Comment (mattpap) [inline examples/models/file/daylight.py:2]: I suppose `Patch` and `Line` should be enclosed in double backticks, or do we have to use the full `:class:` syntax (/cc @bokeh/dev)?.

**Comment by mattpap** [comment]
@ramanshah, when saying "fixes  #NNNN" in the top-level comment, the issue would be closed automatically when this PR is merged. However, this PR only addresses a part of an issue, so a different word is needed. We usually say "addresses #NNNN" instead  in such a situation.

File: examples/models/file/daylight.py (hunk 1 above)
Comment (ramanshah) [reply examples/models/file/daylight.py:1]: That works for me.

File: examples/models/file/daylight.py (hunk 2 above)
Comment (ramanshah) [reply examples/models/file/daylight.py:2]: Good catch!

**Comment by ramanshah** [comment]
Thanks for the quick review! Appreciate your patience - this is my first `bokeh` PR.

File: examples/models/file/daylight.py (hunk 2 above)
Comment (ramanshah) [reply examples/models/file/daylight.py]: I put in the double backticks; let me know if you need a further change.

**Comment by ramanshah** [comment]
@mattpap - I've addressed your above comments. Is there anything else here you feel needs to change before this would be ready to merge?

**Comment by mattpap** [comment]
@ramanshah, thanks for your contribution.

**Pull request merged**